asp_msgs/BoundingBox bbox
---
float32 occupied
float32 free
float32 unknown
